app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationScreen.kt

```diff
@@ -236,11 +236,15 @@ private fun CompleteRegistrationContent(
 
         var showPassword by rememberSaveable { mutableStateOf(false) }
         BitwardenPasswordField(
-            label = stringResource(id = R.string.master_password),
+            label = stringResource(id = R.string.master_password_required)
+                .takeIf { showNewOnboardingUi }
+                ?: stringResource(id = R.string.master_password),
```

Comment:
Could we do this instead?
```kotlin
label = stringResource(
    id = R.string.master_password_required
        .takeIf { showNewOnboardingUi }
        ?: R.string.master_password,
)
```

Reply:
making this change throughout the file for similar places